<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QComboBox" name="mode_chooser_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="output_chooser_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>140</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>271</width>
       <height>118</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="protein_chooser_2"/>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="smile_chooser_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="protein_chooser_1"/>
      </item>
      <item>
       <widget class="QComboBox" name="ligand_chooser_1"/>
      </item>
      <item>
       <widget class="QComboBox" name="mode_chooser"/>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="output_chooser">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="minimizer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MD Minimization</string>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>270</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
